src/osp/Active/SysVehicle.cpp

```diff
@@ -308,14 +306,22 @@ ActiveEnt SysVehicle::part_instantiate(PrototypePart& part,
                     DependRes<ImageData2D> imageData = package.get<ImageData2D>(texName);
                     texRes = AssetImporter::compile_tex(imageData, package);
                 }
-                textureResources.push_back(&(*texRes));
+                textureResources.push_back(texRes);
             }
 
             // by now, the mesh and texture should both exist
-
-            CompDrawableDebug& bBocks
-                    = m_scene.reg_emplace<CompDrawableDebug>(
-                        currentEnt, &(*meshRes), std::move(textureResources), m_shader.get(), 0x0202EE_rgbf);
+            using adera::shader::PhongShaderInstance;
+            Package& glResources = m_scene.get_application().get_gl_resources();
+            PhongShaderInstance si;
```

Comment:
Why bother with the using statement above?

Why not just

adera::shader::PhongShaderInstance si;

?

Reply:
It gets used again on line 321 and makes the line too long, so I figured I'd just do this to shorten both 315 and 321